packages/next-admin/src/components/Breadcrumb.tsx

```diff
@@ -4,7 +4,7 @@ import { useConfig } from "../context/ConfigContext";
 import { ModelIcon } from "../types";
 import ResourceIcon from "./common/ResourceIcon";
 
-export default function Breadcrumb({
+function Breadcrumb({
```

Comment:
```suggestion
export function Breadcrumb({
```

Since you are importing it as a named import elsewhere

So you can remove the default export at the bottom

Reply:
Lets keep it as a default export as most of the other components are using it !

Just change the imports you modified so that the default export is used

Reply:
done, hopefully i didn't miss any!